apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/open_trades/trade_state/states/SellerStateLightning3b.java

```diff
@@ -103,12 +102,17 @@ public void add(BisqEasyOpenTradeMessage message) {
                                 String expectedEncoded = Res.encode("bisqEasy.tradeState.info.buyer.phase3b.tradeLogMessage.ln", peersUserName);
                                 boolean shouldUpdateUI = encodedLogMessage.equals(expectedEncoded);
                                 if (!shouldUpdateUI) {
-                                    String clearedPeersUserName = UserProfileService.getClearedUserName(peersUserName);
-                                    if (!clearedPeersUserName.equals(peersUserName)) {
-                                        String simplifiedExpectedEncoded = Res.encode("bisqEasy.tradeState.info.buyer.phase3b.tradeLogMessage.ln",
-                                                clearedPeersUserName);
-                                        shouldUpdateUI = encodedLogMessage.equals(simplifiedExpectedEncoded);
-                                    }
+                                    /*
+                                     * userName formats may differ between sender and receiver. A userName appears as
+                                     * simple "nickname" if unique in user's storage, or as "nickname [nym-id]" if
+                                     * multiple users share that nickname. Due to persistence state differences, one
+                                     * peer might use the simple format while the other uses the extended format.
+                                     * We therefore compare the encoded message against both the userName and the
+                                     * nickname alone to ensure reliable message matching.
+                                     */
+                                    String simplifiedExpectedEncoded = Res.encode("bisqEasy.tradeState.info.buyer.phase3b.tradeLogMessage.ln",
+                                            bisqEasyOpenTradeChannel.getPeer().getNickName());
+                                    shouldUpdateUI = encodedLogMessage.equals(simplifiedExpectedEncoded);
```

Comment:
I think a more clean way is to just have an `or` expression with checks the nickname version if the username version fails. The Res.encode call is super cheap thus code readability is here preferred.